src/haz3lschool/Grading.re

```diff
@@ -232,6 +232,8 @@ module F = (ExerciseEnv: SchoolExercise.ExerciseEnv) => {
       impl_grading_report: ImplGradingReport.t,
     };
 
+    /* Used in haz3lschool/Gradescope.re and
```

Comment:
I don't think we need to document just the use sites since that can easily be determined in your editor using a project-wide find command, e.g. here for `Grading.mk`